lte/gateway/c/core/oai/tasks/mme_app/mme_app_defs.h

```diff
@@ -30,13 +30,20 @@
 #ifndef FILE_MME_APP_DEFS_SEEN
 #define FILE_MME_APP_DEFS_SEEN
 
-#include "lte/gateway/c/core/oai/lib/itti/intertask_interface.h"
+#include <czmq.h>
 
+#include "lte/gateway/c/core/oai/lib/itti/intertask_interface.h"
```

Comment:
Move  under extern "C". General comment is to move hash library related header files under extern "C".